trunk/src/utest/srs_utest_protocol.cpp

```diff
@@ -3355,14 +3355,14 @@ VOID TEST(ProtocolStackTest, ProtocolAckSizeFlow)
     }
     // recv auto send acked size. #1
     if (true) {
-        SrsCommonMessage* msg = NULL; SrsAutoFree(SrsCommonMessage, msg);
+        SrsCommonMessage* msg = NULL; SrsUniquePtr<SrsCommonMessage> msg_uptr(msg);
```

Comment:
memory leak warning in `srs utest`, check the `utest-regression-blackbox-test`.

```suggestion
        SrsCommonMessage* msg = NULL;
        HELPER_ASSERT_SUCCESS(proto.recv_message(&msg));
        SrsUniquePtr<SrsCommonMessage> msg_uptr(msg)
        
```

Reply:
Fixed.